cpp/tests/structs/utilities_tests.cpp

```diff
@@ -44,9 +47,18 @@ void flatten_unflatten_compare(table_view const& input_table)
 }
 
 using namespace cudf;
+using iterators::no_nulls;
```

Comment:
You are declaring multiple `using iterators::` statements, so just `using iterators;`.

Reply:
Fixed in 5ded23f.